src/components/OverridableAsset.vue

```diff
@@ -8,7 +8,10 @@
   See https://swift.org/CONTRIBUTORS.txt for Swift project authors
 -->
 <template>
-  <ImageAsset v-if="shouldUseAsset" :variants="variants" />
+  <ImageAsset
+    v-if="shouldUseAsset"
+    v-bind="{ variants, loading: null, shouldCalculateOptimalWidth }"
```

Comment:
I'm not 100% convinced that the `loading` override is needed since the assets are still being loaded asynchronously for the first time when the nav containing them is expanded anyway, but I haven't done enough testing to say for sure.